backend/app/src/main/kotlin/io/tolgee/security/jwtTokenExt.kt

```diff
@@ -0,0 +1,9 @@
+package io.tolgee.security
+
+import io.jsonwebtoken.Claims
+
+const val JWT_TOKEN_SUPER_EXPIRATION_CLAIM = "superExpiration"
+
+var Claims.superExpiration: Long?
```

Comment:
Considering we already have a wrapper around `io.jsonwebtoken` stuff, I'm unsure if extending the object like this is a good idea and we should tuck everything under our `JwtToken` wrapper.

Or, we should drop that class altogether and only use the `Claims` object directly.